metal3-dev/run.sh

```diff
@@ -13,12 +15,23 @@ fi
 
 source $(dirname $(dirname $0))/utils.sh
 
+if [ ! -f "ocp/ostest/.openshift_install_state.json" ]; then
```

Comment:
Again this won't work if you cd into the directory first, so it should use SCRIPTDIR (either calculated here or in common.sh)